assets/css/style.css

```diff
@@ -270,8 +270,8 @@ h1, h2, h3, h4, h5, h6 {
   border-color: #24b7a4;
 }
```

Comment:
I am pushing a fix here because while hovering the CNCF link the icon disappears. 
With this fix, the icon stays there while we hover over the link.

```
#header .social-links a:hover:not(.logo-cncf) {
  background: #24b7a4;
  border-color: #24b7a4;
}

#header .social-links a.logo-cncf:hover {
  background-color: #24b7a4;
}
```